src/solve1.F

```diff
@@ -399,7 +399,8 @@ subroutine solve1(nstep,rbufsz,num_soil_layers,                  &
       !$acc present(qa,qten) &
       !$acc present(kmh,kmv,khh,khv,tkea,tketen,nm,dissten,epst) &
       !$acc present(pta,ptten,u2pt,v2pt) &
-      !$acc present(tdiag,qdiag,udiag,vdiag,wdiag,kdiag)
+      !$acc present(tdiag,qdiag,udiag,vdiag,wdiag,kdiag) &
+      !$acc present(uavg,vavg,thavg,qavg,cavg)
```

Comment:
@sjsprecious Why were these arrays added into a present clause here and not in other subroutines where they were also used like sfc_and_turb